nncf/quantization/algorithms/min_max/openvino_backend.py

```diff
@@ -179,7 +155,15 @@ def get_statistic_collector(
         inplace: bool,
         num_samples: int = None,
     ) -> TensorCollector:
-        reduction_axes = OVMinMaxAlgoBackend._get_reduction_axes(nncf_graph, target_point, collector_params)
+        node = nncf_graph.get_node_by_name(target_point.target_node_name)
+        if target_point.is_weight_target_point():
+            assert isinstance(node.layer_attributes, OVLayerAttributes)
+            shape = node.layer_attributes.constant_attributes[target_point.port_id]["shape"]
+            channel_axes = get_weight_channel_axes(node)
+        else:
+            shape = OVMinMaxAlgoBackend._get_activation_shape(target_point, nncf_graph, node)
+            channel_axes = (1,)
```

Comment:
Let's keep original comment here
```suggestion
            channel_axes = (1,) # OpenVINO activations have channel first layout: [N, C, Z, Y, X]
```

Reply:
This comment is actually applicable for all backends, so I removed it